frameworks/g-cloud-9/questions/services/allTheThingsToSee.yml

```diff
@@ -0,0 +1,26 @@
+question: This is question
+question_advice: This is advice to help you answer the question
+name: This is name, for summary tables and slug
+empty_message: this question has not been answered yet
+hint: hint about the answer
+optional: false
+type: checkboxes
+options:
+  - label: Option 1
+    description: >
+       A description of option 1.
+    value: option_one
+  - label: Option 2
+    description: >
+       A description of option 2.
+    value: option_two
+validations:
+   -
+     name: answer_required
+     message: You messed it up you plonker.
+depends:
+  - "on": "lot"
```

Comment:
Thinking back I'm relatively sure that it's quoted because things broke when it wasn't.I'm 99% sure that these got quoted because we did see issues with it being treated as "true" at some point.  I can't remember the details though :/